### DISCUSSION TITLE
Sometimes it hides under a full-screen window when you open it(M4)

### DISCUSSION THREAD

**Discussion by EEErink** [seerge/g-helper#1024] [Q&A] [answered]
title

**Comment by EEErink** [comment]
could it stay on the front?

**Reply by seerge** [reply, marked as answer]
You can check to keep app always on top under Extra

**Reply by EEErink** [reply]
thank u!It was my oversight.

**Comment by EEErink** [comment]
When you open it on the desktop and then open the edge full screen, it will be hidden behind the full screen.
